### PR TITLE
 Added note about non-GET links getting converted to GET links in mailer  Added note about non-GET links getting converted to GET links in mailer templates [ci skip]

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**

### CODE REVIEW

**Comment by prathamesh-sonpatki** [comment]
//cc @senny

rails_guides_

**Comment by senny** [comment]
@prathamesh-sonpatki can you format the note so that it will put the text into the yellow box? I think it's:

> NOTE: ...

**Comment by prathamesh-sonpatki** [comment]
Done. 
rails_guides__

File: guides/source/action_mailer_basics.md [hunk 1]
```diff
@@ -530,6 +530,8 @@ url helper.
 <%= user_url(@user, host: 'example.com') %>
 ```
 
+NOTE: `non-GET` links don't work when added to mailer templates. They are treated as `GET` links.
```
Comment (senny) [inline guides/source/action_mailer_basics.md]: What do you think about?

> as non-GET links require JavaScript and won't work in mailer templates. They will result in normal GET requests.

Comment (prathamesh-sonpatki) [inline guides/source/action_mailer_basics.md]: @senny Should we mention `jQuery UJS` instead of `JavaScript`? Rest of the sentence looks good to me.

Comment (senny) [inline guides/source/action_mailer_basics.md]: @prathamesh-sonpatki yea that's probably better. You can even link the word to the project https://github.com/rails/jquery-ujs

**Comment by prathamesh-sonpatki** [comment]
@senny updated.

**Pull request merged**